src/resources/elements/address-link/address-link.html

```diff
@@ -1,6 +1,10 @@
 <template>
   <div class="addressLink">
-    <a if.to-view="textClickable" ptooltip.to-view="address" click.delegate="gotoEtherscan()">
+    <a
+      if.to-view="textClickable"
+      ptooltip.to-view="address && showTooltip ? address : null"
+      click.delegate="gotoEtherscan()"
```

Comment:
why aren't you using the ethercanlink component?

Reply:
The <etherscanlink> component doesn't have any pop out icon in it so the address link component handles that
![image](https://user-images.githubusercontent.com/2163332/162509008-5b61ad46-749e-49e1-8bf8-afdc3a083258.png)